### ISSUE TITLE
How to prevent tray icon from being created?

### ISSUE THREAD

**Issue by j4hangir** [spyoungtech/ahk#99]
using a simple `win_get` results in a tray icon becoming visible and hiding quickly, and I couldn't find a proper way to disable it without modifying the templates, esp. `base.ahk`

**Comment by spyoungtech** [comment]
Hi @j4hangir thanks for reporting this. I thought there was already a way to do this with the `ahk.directives.NoTrayIcon` directive. If so, it sounds like this may be a documentation task. Alternatively, it may be a matter of making directives passable to the AHK class init function.

**Comment by spyoungtech** [comment]
Hi @j4hangir this is now implemented in version 0.11.1

```python
from ahk import AHK
from ahk.directives import NoTrayIcon

ahk = AHK(directives=[NoTryIcon])
```
